### DISCUSSION TITLE
Isn't the pronunciation supposed to be "Hurfbaaz" ?

### DISCUSSION THREAD

**Discussion by RayyanNafees** [harfbuzz/harfbuzz#4342] [Ideas] [answered]
# The Arabic Text in the logo suggests then name to be "Hurf" (character) "baaz" (master).. Hurfbaaz seems a plausible name

# What does Harfbuzz even mean ?

**Comment by alerque** [comment, marked as answer]
See https://harfbuzz.github.io/why-is-it-called-harfbuzz.html.
